Patch by MSP-Greg
Originally tried to see if it improved readline
diff --git a/ext/readline/extconf.rb b/ext/readline/extconf.rb
index 7bba386540..5266e07439 100644
--- a/ext/readline/extconf.rb
+++ b/ext/readline/extconf.rb
@@ -83,9 +83,9 @@ def readline.have_type(type)
 readline.have_var("rl_point")
 readline.have_var("rl_char_is_quoted_p")
 # workaround for native windows.
-/mswin|bccwin|mingw/ !~ RUBY_PLATFORM && readline.have_var("rl_event_hook")
-/mswin|bccwin|mingw/ !~ RUBY_PLATFORM && readline.have_var("rl_catch_sigwinch")
-/mswin|bccwin|mingw/ !~ RUBY_PLATFORM && readline.have_var("rl_catch_signals")
+/mswin|bccwin/ !~ RUBY_PLATFORM && readline.have_var("rl_event_hook")
+/mswin|bccwin/ !~ RUBY_PLATFORM && readline.have_var("rl_catch_sigwinch")
+/mswin|bccwin/ !~ RUBY_PLATFORM && readline.have_var("rl_catch_signals")
 readline.have_var("rl_pre_input_hook")
 readline.have_var("rl_special_prefixes")
 readline.have_func("rl_cleanup_after_signal")
